### PR TITLE
i18n for content pages

### LINKED ISSUE #775: All blog posts have content of join-raic
In the current staging, all the blog posts have the content of the join-raic post:

https://staging-aiid.netlify.app/blog/representation-and-imagination
https://staging-aiid.netlify.app/blog/the-first-taxonomy-of-ai-incidents

### PR DESCRIPTION
**Pull request by cesarvarela**
also closes #775

### CODE REVIEW

File: site/gatsby-site/src/components/forms/QuickAddForm.js [hunk 1]
```diff
@@ -55,7 +58,7 @@ const QuickAddForm = ({ className = '' }) => {
           message: (
             <>
               {'Report successfully added to review queue. It will appear on the  '}
```
Comment (lmcnulty) [inline site/gatsby-site/src/components/forms/QuickAddForm.js:60]: Is there a reason why the toast messages aren't wrapped in `t()` ?

Comment (cesarvarela) [reply site/gatsby-site/src/components/forms/QuickAddForm.js:60]: I forgot about them. I will create another pr for those.

File: site/gatsby-site/blog/join-raic/index.es.mdx [hunk 2]
```diff
@@ -5,7 +5,7 @@ metaDescription: ''
 date: '2022-03-29'
 image: ../../src/images/the_place.jpg
 author: Responsible AI Collaborative
```
[the remark this answers is not in the document]
Comment (cesarvarela) [reply site/gatsby-site/blog/join-raic/index.es.mdx:7]: Thanks totally missed that. https://github.com/responsible-ai-collaborative/aiid/issues/803

File: site/gatsby-site/content/research/1-criteria/index.es.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+# ¿Qué no es un incidente de IA?
+
+Varias situaciones comunes _no_ cumplen con los criterios anteriores y no deben registrarse como incidentes en la base de datos de incidentes de AI. Éstos incluyen:
+
+| Tipo | Ejemplo |
+|----------------------------------------------------------------- -------------------------------------------------- -------------------------------------------------- --|----------------------------------------------- -------------------------------------------------- -------------------------------------------------- -------------------------------------------------- -----------------------|
+| Los sistemas de IA funcionan mal en control o tentornos de simulación, incluidos entornos simulados | [Las ligeras modificaciones de letreros de calles pueden engañar por completo a los algoritmos de aprendizaje automático](https://spectrum.ieee.org/cars-that-think/transportation/sensors/slight-street-sign-modifications-can-fool-machine-learning-algorithms ) |
```
[the remark this answers is not in the document]
Comment (cesarvarela) [reply site/gatsby-site/content/research/1-criteria/index.es.mdx]: Google translate messed up the formatting. Fixed.

**Review by lmcnulty — APPROVED** [review]

**Pull request merged**